presentation/main_view.py

```diff
@@ -1,27 +1,38 @@
 from datetime import datetime
-from typing import Dict, List
 
 import dash
 import dash_core_components as dcc
 import dash_html_components as html
-from dash.dependencies import Input, Output
 
 from domain.config import Config
 from domain.model import MeshResults
-from presentation.matrix_view import make_mesh_test_matrix_layout
 
 
 def make_page_layout(mesh: MeshResults, results_timestamp: datetime, config: Config) -> html.Div:
     timestamp = results_timestamp.strftime("%H:%M:%S")
-    header = f"Demo WebApp main page - test results timestamp {timestamp}"
+    header = "Synthetics SLA Dashboard WebApp"
```

Comment:
```suggestion
    header = "Synthetics SLA Dashboard"
```

Reply:
user knows he uses web browser; less is more

Reply:
Agreed ... and even less is even better:

```header = "SLA Dashboard"```